airflow/www/views.py

```diff
@@ -1351,12 +1351,9 @@ def clear(self):
 
     @expose('/all_tasks', methods=['GET'])
     @login_required
+    @wwwutils.gzipped
```

Comment:
just importing the lib is all you needed?

Reply:
using a decorator, yeah. airflow team already built it for us!